src/compositions/ChooseLanguage/ChooseLanguage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import LanguageDropdown from '../../components/LanguageDropdown/LanguageDropdown';
+
+import './ChooseLanguage.css';
+
+const ChooseLanguage = ({ language, setLanguage }) => {
```

Comment:
This is where you would name the div that is parent to className titleText ChooseLanguage, then in the .css file everything should start with .ChooseLanguage, this allows for css scoping and for css classes to only be active, when that component is active. Currently in the css file those classes are are global and not scoped to this component. When you add the component name as the parent className, not only does it help with debugging but allows for VERY powerful and versitile css scoping.